src/editors/containers/VideoEditor/components/VideoSettingsModal/components/TranscriptWidget/index.jsx

```diff
@@ -113,7 +115,8 @@ export const TranscriptWidget = ({
             <FormattedMessage {...messages.addFirstTranscript} />
           </>
         )}
-        <Button iconBefore={FileUpload} onClick={() => { console.log('adding file'); }} variant="link">
+        <FileInput fileInput={fileInput} acceptedFiles=".srt" />
```

Comment:
Did you do any CSS to hide this fileinput, or does it show up on the UI?

Reply:
Just the one quick question

Reply:
otherwise looks good

Reply:
No CSS and it doesn't appear in the UI. The button controls the ref that provides the functionality of an `input` element. I am following the same structure that is used in the `SelectImageModal`

Reply:
Ok Awesome! I will give the thumb and we can merge this in and i will rebase off it.